src/context/Discussions/DiscussionProvider.tsx

```diff
@@ -143,7 +144,8 @@ const DiscussionProvider: FC<DiscussionProviderProps> = ({ children }) => {
           message: post,
         },
       },
-    });
+      onError: handleError,
+    })) as Promise<FetchResult<void>>;
```

Comment:
Here I'm casting a `Promise<FetchResult<PostDiscussionCommentMutation, Record<string, any>, Record<string, any>>>` to `Promise<FetchResult<void>>` because I don't want to have to drag those imports, `Record`, `PostDiscussionCommentMutation` all the way up the dependent classes..
I only need to access the `FetchResult` to see if it has errors or not. But I'm not sure if this is the cleanest way.

Reply:
I can't do that because I need the FetchResult outside to see if there were any errors. 
I just don't need the rest of the _Fetched Result_ properties.

Reply:
Very good one. Thanks